R/code_agreements.R

```diff
@@ -61,40 +61,40 @@ code_agreements <- function(dataset = NULL, title, date) {
   out <- vector(mode = "character", length = length(title)) # initialize vector
   # for agreements (A) where abbreviation is known and
   # bilateral agreement is made subsequently
-  qID <- ifelse(!is.na(abbrev) & (type == "A") & !is.na(parties),
+  treatyID <- ifelse(!is.na(abbrev) & (type == "A") & !is.na(parties),
```

Comment:
These are lintr notes that happen because we have upper case letters in some of the variable names (e.g. treatyID). Normally, snake_case style requires that the first letter of the variable is not upper case and that all spaces are filled with "_". However, it seems that for lintr snake_case is more strict. We can leave as is for now, this not a big deal. Thank you.